x-pack/plugins/monitoring/public/components/logstash/pipeline_viewer/views/__test__/__snapshots__/detail_drawer.test.js.snap

```diff
@@ -7,14 +7,24 @@ exports[`DetailDrawer component If vertices shows basic info and no stats for if
 >
   <EuiFlyoutHeader>
     <EuiFlexGroup
-      alignItems="center"
+      alignItems="baseline"
       component="div"
       direction="row"
-      gutterSize="l"
+      gutterSize="s"
       justifyContent="flexStart"
       responsive={true}
       wrap={false}
     >
+      <EuiFlexItem
+        component="div"
+        grow={false}
+      >
+        <img
```

Comment:
Maybe it is better to use an EuiIcon here? Along with the other areas using ad-hoc omg tags? But that's not a blocking item IMHO and is perhaps better in a followup PR anyway.

Reply:
I'll create an issue to track it. These changes are indirect to the scope of this PR, it's based on some cleanup that Dave did after he updated the styles.